### PR TITLE
feat(AAiT-web-2): Added About Us Page

### PR DESCRIPTION
**Pull request by yakin-ts**
Implemented an About us page. Responsive design and minor fixes on UI like images will be added next.

### CODE REVIEW

**Comment by yakin-ts** [comment]
I have updated the files accordingly.

File: aait/web/group-2/app/about-us/page.tsx [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+            id: '5',
+            icon: '/images/Ellipse-5.png',
+            title: 'Bi-weekly 1:1s',
+            body: 'In 1:1s, we can talk about anything that matters; clearly no boundaries. The more we speak our minds without a filter, the better for the team.'
+        },
+
+
+    ]
+};
+
```
[the remark this answers is not in the document]
Comment (yakin-ts) [reply aait/web/group-2/app/about-us/page.tsx]: Done.

File: aait/web/group-2/app/about-us/page.tsx [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+export default function AboutUs() {
```
[the remark this answers is not in the document]
Comment (yakin-ts) [reply aait/web/group-2/app/about-us/page.tsx]: Done

**Review by yakin-ts — COMMENTED** [review]
reviewed and done.

**Review by zelalemyetsedaw — DISMISSED** [review]
lGTM

**Pull request closed without merging**